packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/SelectedItemDefault.tsx

```diff
@@ -24,7 +24,7 @@ export const SelectedItemDefault: (props: IPeoplePickerItemProps) => JSX.Element
       <div className='ms-PickerItem-content'>
         <Persona
           { ...item }
-          presence = { item.presence ? item.presence : PersonaPresence.online }
+          presence = { item.presence != null ? item.presence : PersonaPresence.online }
           size={ PersonaSize.extraSmall }
```

Comment:
Thanks a bunch fo fixing this. Could you also set PersonaPresence.online to PersonaPersence.none? If item.presence is null or undefined it shouldn't show them as online either..